main.nf

```diff
@@ -394,6 +396,9 @@ process markDuplicates {
         ASSUME_SORTED=true \\
         PROGRAM_RECORD_ID='null' \\
         VALIDATION_STRINGENCY=LENIENT
+
+    #Printing out version number to standard out
+    echo "File name: $bam_markduplicates MarkDuplicates version "$(java -Xmx2g -jar $PICARD_HOME/picard.jar  MarkDuplicates --version 2>&1)
```

Comment:
Are you saying that it should be referencing picard instead? Or that it shoud reference picard?